images/scripts/fedora.setup

```diff
@@ -190,32 +194,34 @@ fi
 
 # don't use a bootstrap chroot to save time and space; everything is offline, and we always just do exactly one build
 echo "config_opts['use_bootstrap'] = False" >>/etc/mock/site-defaults.cfg
-su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(/var/lib/testvm/build-deps.sh "${ID} ${VERSION_ID}")"
+if [ "fedora-eln" != "$IMAGE" ]; then
```

Comment:
This is now an ugly special case. I'd rather you taught build-deps.sh to handle eln properly, with `fedora*eln) ;;` You almost had it correct before, but the argument is not separated with `-` but with ` `

Reply:
... and I suppose you still actually do want to build stuff on the eln image, so you still need build-deps.sh?